src/inference/src/dev/threading/istreams_executor.cpp

```diff
@@ -265,7 +261,7 @@ void IStreamsExecutor::Config::update_executor_config() {
         }
     }
 
-    if (_cpu_pinning) {
+    if (_cpu_pinning || _cpu_reservation) {
```

Comment:
If user set cpu_reservation only, change cpu_pinning to yes internally.
If user set both cpu_reservation and cpu_pinning, keep user setting.

Reply:
Ok, done.